python/ray/serve/schema.py

```diff
@@ -1029,7 +1029,7 @@ class ServeInstanceDetails(BaseModel, extra=Extra.forbid):
     This is the response JSON schema for v2 REST API `GET /api/serve/applications`.
     """
 
-    controller_info: ServeActorDetails = Field(
+    controller_info: Optional[ServeActorDetails] = Field(
```

Comment:
Is this already getting set? I don't see anything in the diff that actually sets this to a non-`None` value.

Reply:
It gets set in `controller.get_serve_instance_details()` (the dashboard makes a call to the controller to get the metadata).